decidim-meetings/app/models/decidim/meetings/meeting.rb

```diff
@@ -12,7 +12,9 @@ class Meeting < Meetings::ApplicationRecord
       belongs_to :scope, foreign_key: "decidim_scope_id", class_name: Decidim::Scope
       belongs_to :category, foreign_key: "decidim_category_id", class_name: Decidim::Category
       has_one :organization, through: :feature
-      validates :title, presence: true
+
+      validates :title, :feature, presence: true
+      validate :feature_manifest_matches
```

Comment:
What about moving this logic to a concern with a `validate_feature :meetings` macro or something similar? Then we can use it in other features as well.